pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1760,7 +1760,13 @@ func makePod(taskRun *v1beta1.TaskRun, task *v1beta1.Task) (*corev1.Pod, error)
 		return nil, err
 	}
 
-	return podconvert.MakePod(context.Background(), images, taskRun, task.Spec, kubeclient, entrypointCache, true)
+	builder := podconvert.PodBuilder{
```

Comment:
nit: it bothers me it's not named the same as above :joy_cat: *but that's definitely a nit* :joy:

Reply:
hm, i'm open to changing the naming, no big deal, but i'm not 100% clear what name you want - right now it's `podconvert.Builder` after golangci-lint corrected me.  What's your preferred name?

Reply:
I was speaker about the variable name `builder` (and above `podBuilder`) :blush: